src/cmd/flux.c

```diff
@@ -97,13 +94,8 @@ static optparse_t * setup_optparse_parse_args (int argc, char *argv[])
     if (e != OPTPARSE_SUCCESS)
         log_msg_exit ("optparse_add_option_table() failed");
 
-    // Remove automatic `--help' in favor of our own usage() from above
-    e = optparse_remove_option (p, "help");
-    if (e != OPTPARSE_SUCCESS)
-        log_msg_exit ("optparse_remove_option (\"help\")");
-    e = optparse_add_option (p, &helpopt);
-    if (e != OPTPARSE_SUCCESS)
-        log_msg_exit ("optparse_add_option (\"help\")");
+    // Disable automatic `--help' in favor of our own usage() from above
+    optparse_set (p, OPTPARSE_OPTION_CB, "help", NULL);
 
```

Comment:
Though it is almost impossible to have an error, should we check error here just for completeness? (And in case code  gets duplicated or used as an example)